src/iterator.cr

```diff
@@ -1377,4 +1376,96 @@ module Iterator(T)
       self
     end
   end
+
+  # Returns an iterator for each chunked elements where each chunk
+  # ends right **before** the given block's value is _truthy_.
```

Comment:
either bold before above or remove it here

Reply:
Done!